front/app/modules/commercial/flag_inappropriate_content/index.tsx

```diff
@@ -29,6 +30,19 @@ const configuration: ModuleConfiguration = {
     ) => {
       return <RemoveFlagButton {...props} />;
     },
+    'app.modules.commercial.moderation.admin.containers.ModerationRow.content': ({
+      inappropriateContentFlagId,
+    }) => {
+      if (inappropriateContentFlagId) {
```

Comment:
Doing it like this (optional inappropriateContentFlagId and component rendering) because this outlet could be used by other places of the code base, so we might not want to render this particular component all the time.

Reply:
Looks good to me! :)